javascript.html.markdown

```diff
@@ -603,31 +603,39 @@ const pi = 3.14;
 
 pi = 4.13; // You cannot do this.
 
-// There is a new syntax for functions in ES6 known as "lambda syntax".
+// There is a new syntax for functions in ES6 known as "arrow function"
+// (lambda syntax in many others languages).
 // This allows functions to be defined in a lexical scope like with variables
 // defined by const and let. 
+// By default, a single statement without '{ }' defined will be the return statement, hence the `return` keyword is not required. 
 
-const isEven = (number) => {
-    return number % 2 === 0;
-};
+const isOdd = number => number % 2 !== 0;
+
+// But, if '{ }' are used, then a `return` is required.
 
-isEven(7); // false
+const isEven = number => { return number % 2 == 0 }
 
 // The "equivalent" of this function in the traditional syntax would look like this:
 
-function isEven(number) {
-    return number % 2 === 0;
+function isOdd(number) {
+    return number % 2 !== 0;
```

Comment:
These are not equivalent, they're completely different functions. Let's choose one (`isEven` is simpler) and use that multiple times.